imageroot/bin/module-dump-state

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+# clean container files on exit
+cleanup () {
+    podman exec mariadb rm -rf /tmp/db_backup
+    podman exec asterisk rm -f /var/lib/asterisk/db/astdb.sqlite3.dump
+}
+
+trap cleanup EXIT
+
+## MariaDB
+# exec mariabackup to save database in /tmp/db_backup
+podman exec mariadb \
+    mariabackup \
+        --backup \
+        --target-dir=/tmp/db_backup \
+        --user=root \
+        --password="${MARIADB_ROOT_PASSWORD}"
+# copy db_backup in local filesystem
+podman cp mariadb:/tmp/db_backup db_backup
```

Comment:
I'm not sure this is really needed.
We just dump to local filesystem inside other modules: https://github.com/NethServer/ns8-mattermost/blob/main/imageroot/bin/module-dump-state#L11

Reply:
The dump with `mariabackup` is different, it'll backup the entire database, andthe output is a folder

Reply:
:scream: